valhalla/jawn/src/lib/shared/request/request.ts

```diff
@@ -89,6 +89,7 @@ export async function getRequests(
       THEN '{"helicone_message": "request body too large"}'::jsonb
       ELSE request.body::jsonb
     END AS request_body,
+    request.country_code as country_code,
```

Comment:
Please add to:

`getRequests`
&
`getRequestsCached`

Also it parses to `HeliconeRequest` object. If we want this returned to FE, we have to add country_code to `HeliconeRequest`.

Also, please add to

`web/lib/api/request/request.ts`

That will allow country_code to return to graphQL